windows-installer/Package.wxs

```diff
@@ -46,13 +45,13 @@
 			DllEntry="WixQuietExec"
 			Execute="deferred"
 			Impersonate="yes"
-			Return="check"
+			Return="ignore"
```

Comment:
I have also changed the Return to ignore for `ideasy -f install` because right now the installations fails when ideasy already exists and that can be confusing. But you can just revert this if you want to keep the check.

Reply:
I would rather consider changing the installation mechanism inside IDEasy to be more robust and idempotent.
```suggestion
			Return="check"
```